modules/node_groups/README.md

```diff
@@ -27,7 +27,9 @@ The role ARN specified in `var.default_iam_role_arn` will be used by default. In
 | disk\_encrypted | Whether the root disk will be encrypyted. Requires `create_launch_template` to be `true` and `disk_kms_key_id` to be set | bool | false |
 | disk\_kms\_key\_id | KMS Key used to encrypt the root disk. Requires both `create_launch_template` and `disk_encrypted` to be `true` | string | "" |
 | disk\_size | Workers' disk size | number | Provider default behavior |
-| disk\_type | Workers' disk type. Require `create_launch_template` to be `true`| number | `gp3` |
+| disk\_type | Workers' disk type. Require `create_launch_template` to be `true`| string | Provider default behavior |
```

Comment:
updated since we aren't setting it to `gp3`, and the default by provider is `gp2`
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/launch_template#volume_type

Reply:
I dont think we should change default, what is more this `README.md` is auto generated using terraform-docs so we shouldn`t update it